wire-ios/Wire-iOS Tests/Mocks/UserSessionMock.swift

```diff
@@ -55,15 +68,19 @@ final class UserSessionMock: UserSession {
     }
 
     func enqueue(_ changes: @escaping () -> Void) {
-        fatalError("not implemented")
+        changes()
     }
 
     func enqueue(_ changes: @escaping () -> Void, completionHandler: (() -> Void)?) {
         fatalError("not implemented")
     }
 
-    func addUserObserver(_ observer: ZMUserObserver, for: UserType) -> NSObjectProtocol? {
-        fatalError("not implemented")
+    func addUserObserver(_ observer: ZMUserObserver, for user: UserType) -> NSObjectProtocol? {
+        return nil
+    }
+
+    func addConversationListObserver(_ observer: WireDataModel.ZMConversationListObserver, for list: ZMConversationList) -> NSObjectProtocol {
+        return NSObject()
```

Comment:
I guess you don't really assert anything in your tests with this method

Reply:
Not in this case, no.